examples/run_var_tau.py

```diff
@@ -3,188 +3,159 @@
+        config_dict["deadtime_us"]
+        + config_dict["acq_time_ms"] * 1e3 * (1.0 / 8.0)
+        + tau_adjust
+    )
     print("***")
-    print("INDEX %d - TAU %f"%(index,tau))
+    print("INDEX %d - TAU %f" % (index, tau))
     print("***")
-    SpinCore_pp.configureTX(adcOffset, carrierFreq_MHz, tx_phases, amplitude, nPoints)
-    acq_time = SpinCore_pp.configureRX(SW_kHz, nPoints, nScans, nEchoes, nPhaseSteps) #ms
-    acq_params['acq_time_ms'] = acq_time
-    SpinCore_pp.init_ppg();
+    SpinCore_pp.configureTX(
+        config_dict["adcOffset"],
+        config_dict["carrierFreq_MHz"],
+        tx_phases,
+        config_dict["amplitude"],
+        nPoints,
+    )
+    acq_time = SpinCore_pp.configureRX(
+        config_dict["SW_kHz"],
+        nPoints,
+        config_dict["nScans"],
+        config_dict["nEchoes"],
+        nPhaseSteps,
+    )  # ms
+    SpinCore_pp.init_ppg()
     if phase_cycling:
-        SpinCore_pp.load([
-            ('marker','start',1),
-            ('phase_reset',1),
-            ('delay_TTL',deblank),
-            ('pulse_TTL',p90,'ph1',r_[0,1,2,3]),
-            ('delay',tau),
-            ('delay_TTL',deblank),
-            ('pulse_TTL',2.0*p90,'ph2',r_[0,2]),
-            ('delay',deadtime),
-            ('acquire',acq_time),
-            ('delay',repetition),
-            ('jumpto','start')
-            ])
-    if not phase_cycling: 
-        SpinCore_pp.load([
-            ('marker','start',nScans),
-            ('phase_reset',1),
-            ('delay_TTL',deblank),
-            ('pulse_TTL',p90,0.0),
-            ('delay',tau),
-            ('delay_TTL',deblank),
-            ('pulse_TTL',2.0*p90,0.0),
-            ('delay',deadtime),
-            ('acquire',acq_time),
-            ('delay',repetition),
-            ('jumpto','start')
-            ])
-    SpinCore_pp.stop_ppg();
+        SpinCore_pp.load(
+            [
+                ("marker", "start", 1),
+                ("phase_reset", 1),
+                ("delay_TTL", config_dict["deblank_us"]),
+                ("pulse_TTL", config_dict["p90_us"], "ph1", r_[0, 1, 2, 3]),
+                ("delay", tau),
+                ("delay_TTL", config_dict["deblank"]),
+                ("pulse_TTL", 2.0 * config_dict["p90_us"], "ph2", r_[0, 2]),
+                ("delay", config_dict["deadtime"]),
+                ("acquire", acq_time),
+                ("delay", config_dict["repetition_us"]),
+                ("jumpto", "start"),
+            ]
+        )
+    if not phase_cycling:
+        SpinCore_pp.load(
+            [
+                ("marker", "start", config_dict["nScans"]),
+                ("phase_reset", 1),
+                ("delay_TTL", config_dict["deblank"]),
+                ("pulse_TTL", config_dict["p90_us"], 0.0),
+                ("delay", tau),
+                ("delay_TTL", config_dict["deblank_us"]),
+                ("pulse_TTL", 2.0 * config_dict["p90_us"], 0.0),
+                ("delay", config_dict["deadtime_us"]),
```

Comment:
similar to t1cpmg this is the only script that uses a var tau ppg. So I keep the ppg in the script rather than making a new spincore_pp/ppg function